### PR TITLE
added step by step section

### PR DESCRIPTION
**Pull request by snork-alt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,28 +14,44 @@
 
 ## Overview
 
-Dozer is a **real-time data platform for building, deploying and mantaining data products.**
+Dozer is a **data platform for building, deploying and mantaining real-time data products.**
```
Comment (Jesse-Bakker) [inline README.md:17]: `mantaining` -> `maintaining`

File: README.md [hunk 2]
```diff
@@ -14,28 +14,44 @@
...
+![Screenshot](./images/dozer_live_screen2.png)
+
+### ③ Deploy
+Dozer applications can be self-hosted or deployed in the cloud with a single command. Dozer Cloud (coming soon) provides self-healing and monitoring capabilities, making sure your APIs are always avaialble.
 
-Following the local development and testing, Dozer Apps can be self-hosted or instantly deployed to Dozer Cloud (coming soon).
 
 ## Supported Sources and Tranformation Engines
-Dozer currently supports multiple databases, data warehouses and object store as a source. Whenever possible, Dozer leverages real-time streaming to keep data always fresh. 
+Dozer currently supports a variety of source databases, data warehouses and object store. Whenever possible, Dozer leverages Change Data Capture (CDC) to keep data always fresh. Alternatively, periodic polling is used.
```
Comment (Jesse-Bakker) [inline README.md:49]: `object store` -> `object stores`

Comment (Jesse-Bakker) [inline README.md:49]: `Alternatively,` -> `For sources that do not support CDC,` ?

**Review by v3g42 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -14,28 +14,44 @@
...
+![Screenshot](./images/dozer_live_screen1.png)
 
-![Screenshot](./images/dozer_live_screen.png)
+### ② Test
+Dozer can run the entire infrastructure locally. You can inspect data flowing in in real time or use the built-it API explorer to query data through REST and gRPC. Dozer Live explorer also provides ready-made samples to integrate results into your front-end applications.
+
+![Screenshot](./images/dozer_live_screen2.png)
+
+### ③ Deploy
+Dozer applications can be self-hosted or deployed in the cloud with a single command. Dozer Cloud (coming soon) provides self-healing and monitoring capabilities, making sure your APIs are always avaialble.
```
Comment (Jesse-Bakker) [inline README.md:45]: `avaialble` -> `available`

**Review by Jesse-Bakker — COMMENTED** [review]
One typo, then LGTM

**Pull request merged**